src/blocks/ActionBlock/action.css

```diff
@@ -0,0 +1,15 @@
+/* basic styling, can be replaced */
```

Comment:
Should this file be here, in the common container block for all actions, or should we consider this a core style and move it to `src/assets/action.css`?